src/lang/en-US.json

```diff
@@ -338,8 +338,8 @@
   "lane_role_3": "Off",
   "lane_role_4": "Jungle",
 
-  "lane_1": "Bottom",
-  "lane_2": "Middle",
+  "lane_1": "Bot",
```

Comment:
Do you mean for lane_2? It looks like that's the only one defined elsewhere as something else